modules/tensor_mechanics/src/kernels/GeneralizedPlaneStrainOffDiag.C

```diff
@@ -56,15 +56,16 @@ GeneralizedPlaneStrainOffDiag::GeneralizedPlaneStrainOffDiag(const InputParamete
                        : 0),
     _temp_var(isParamValid("temperature")
                   ? &_subproblem.getVariable(_tid, getParam<NonlinearVariableName>("temperature"))
-                  : NULL)
+                  : NULL),
+    _num_disp_var(getParam<std::vector<NonlinearVariableName>>("displacements").size())
```

Comment:
Maybe we should define _nl_vnames_ as member variable and assign its size to __num_disp_var_

Reply:
We could do that, but I'm not sure it's better.